schemas/snuba-queries.v1.schema.json

```diff
@@ -127,7 +127,33 @@
           "type": ["integer", "null"]
         },
         "stats": {
-          "type": "object"
+          "type": "object",
+          "properties": {
+            "final": {
+              "type": "boolean"
+            },
+            "cache_hit": {
+              "type": "integer"
+            },
+            "sample": {
+              "type": "number"
+            },
+            "max_threads": {
+              "type": "integer"
+            },
+            "clickhouse_table": {
+              "type": "string"
+            },
+            "query_id": {
+              "type": "string"
+            },
+            "is_duplicate": {
```

Comment:
same for this one

Reply:
Same as the other one, it's not required by default